### PR TITLE
[Tech] HR-modal window Test Assignment

### PR DESCRIPTION
**Pull request by kseniyaLine**
CoachTestVerification modal window + HR modal window to assign a test

### CODE REVIEW

File: src/components/Employees/EmployeesTable/HRmodalWindows/HRmodalWindow.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import '../../../../styles/modal.scss';
+import CloseIcon from '@material-ui/icons/Close';
+import {IconButton, FormControl, InputLabel, MenuItem, Select} from '@material-ui/core';
+import PropTypes from 'prop-types';
+import './HRmodalWindow.scss';
+import Button from '@material-ui/core/Button';
+
+export const HRmodalWindow = ({name, handleClose}) => {
+
+  const englishLevel = ['Beginner(A1)', 'Elementary(A2)', 'Intermediate(B1)', 'Upper Intermediate(B2)', 'Advanced(C1)', 'Proficiency(C2)'];
```
Comment (fshovchko) [inline src/components/Employees/EmployeesTable/HRmodalWindows/HRmodalWindow.jsx]: You can move filter array to constants/constants.js

File: src/components/Employees/EmployeesTable/HRmodalWindows/HRmodalWindow.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      </IconButton>
+      {modalBody}
+    </div>);
+};
+
+HRmodalWindow.propTypes =
+  {
+    name: PropTypes.string,
+    handleClose: PropTypes.func
+  };
```
Comment (fshovchko) [inline src/components/Employees/EmployeesTable/HRmodalWindows/HRmodalWindow.jsx]: File name doesn't specify which modal window it is. Rename it to TestAssignment or something like that

File: src/components/Employees/EmployeesTable/HRmodalWindows/HRmodalWindow.scss [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  }
+
+  .MuiButton-root {
+    height: 27px;
+    width: 90px;
+    margin-left: 337px;
+    margin-right: 5px;
+    margin-top: 30px;
+  }
+}
```
Comment (fshovchko) [inline src/components/Employees/EmployeesTable/HRmodalWindows/HRmodalWindow.scss]: File name doesn't specify which modal window it is. Rename it to TestAssignment or something like that

File: src/components/Test/CoachTestVerification/CoachTestVerification.jsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        </div>
+      </div>
+    </Modal>
+  );
+};
+
+CoachTestVerification.propTypes =
+  {
+    status: PropTypes.bool,
+  };
```
Comment (fshovchko) [inline src/components/Test/CoachTestVerification/CoachTestVerification.jsx]: File name doesn't specify which modal window it is. Rename it

File: src/components/Employees/EmployeesTable/EmployeesTable.jsx [hunk 5]
```diff
@@ -37,11 +49,38 @@ export const EmployeesTable = (props) => {
...
+    setOpen(false);
+  };
+
+  const modals = [
+    <HRmodalWindow key={0} name={name} handleClose={handleClose}/>,
+  ];
+
+
   return (
     <div>
```
Comment (fshovchko) [inline src/components/Employees/EmployeesTable/EmployeesTable.jsx:70]: This div is useless. Remove it

File: src/components/Employees/EmployeesTable/HRmodalWindows/HRmodalWindow.scss [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+        font-size: 18px;
+      }
+
+      .priority-selector {
+        width: 200px;
+        font-size: 5px;
+
+        .sel {
+          height: 29px;
+        }
```
Comment (fshovchko) [inline src/components/Employees/EmployeesTable/HRmodalWindows/HRmodalWindow.scss]: This selector name is weird. Please rename it

**Review by mtalstykh-exadel — DISMISSED** [review]

**Review by fshovchko — DISMISSED** [review]

**Review by mtalstykh-exadel — APPROVED** [review]

**Review by fshovchko — APPROVED** [review]

**Pull request merged**